cmd/package-operator-manager/bootstrap/init.go

```diff
@@ -223,19 +222,18 @@ func (init *initializer) config() *runtime.RawExtension {
 func (init *initializer) crdsFromPackage(ctx context.Context) (
 	crds []unstructured.Unstructured, err error,
 ) {
-	files, err := init.pullImage(ctx, init.selfBootstrapImage)
+	rawPkg, err := init.pullImage(ctx, init.selfBootstrapImage)
```

Comment:
so .... I see the test below in the other file that checks this function. any idea why codecov does not accept the coverage of this?